src/calendardata.h

```diff
@@ -49,6 +49,7 @@ struct EventOccurrence {
     QDateTime recurrenceId;
     QDateTime startTime;
     QDateTime endTime;
+    bool eventAllDay;
```

Comment:
Well, my bad. I don't think this should be optional. Because otherwise, this CalendarData::EventOccurrence has no way to tell if the occurrence is all day or not. Which can create strange things if we start to detect all day status from startTime and endTime (which is actually what is done at the moment in the dailyEventOccurrences() as far as I know). So I added the proper initialisation in the utils also. I checked that there is no instanciation elsewhere (if I'm not mistaken).